AppCenter/AppCenter/MSConstants+Flags.h

```diff
@@ -5,7 +5,10 @@
 
 typedef NS_OPTIONS(NSUInteger, MSFlags) {
   MSFlagsNone = (0 << 0),                // => 00000000
-  MSFlagsPersistenceNormal = (1 << 0),   // => 00000001
-  MSFlagsPersistenceCritical = (1 << 1), // => 00000010
-  MSFlagsDefault = MSFlagsPersistenceNormal
+  MSFlagsNormal = (1 << 0),               // => 00000001
+  MSFlagsCritial = (1 << 1),              // => 00000010
+  MSFlagsPersistenceNormal = MSFlagsNormal,
```

Comment:
Is there a way to mark it as deprecated?

Reply:
sure！